22/alpine3.19/Dockerfile

```diff
@@ -45,14 +45,12 @@ RUN addgroup -g 1000 node \
       141F07595B7B3FFE74309A937405533BE57C7D57 \
       74F12602B6F1C4E913FAA37AD3A89613643B6201 \
       DD792F5973C6DE52C432CBDAC77ABFA00DDBF2B7 \
-      61FC681DFB92A079F1685E77973F295594EC4689 \
```

Comment:
This is the only changed file for v22 in this PR, is this intentional?

Reply:
Thanks, that was a mistake, I rebased it out